### PR TITLE
Cloud users are able to request the audit log

### PR DESCRIPTION
**Pull request by ganezasan**
# Description
Update the description that the Server users are able to use the audit log feature via the UI. Add how to get an audit log from the cloud customers.

# Reasons
The current description is mentioned only the server users are able to use the audit log. But the cloud users also are able to request audit logs through Support. And a customer asked whether they can request an audit log or not.

### CODE REVIEW

File: jekyll/_cci2/security.md [hunk 1]
```diff
@@ -59,9 +59,10 @@ A few different external services and technology integration points touch Circle
 {% endraw %}
 
 ## Audit Logs
-The Audit Log feature is only available for CircleCI installed on your servers or private cloud. 
+Server customers who installed CircleCI on your environments are able to use the audit log feature on UI.
```
Comment (rosieyohannan) [inline jekyll/_cci2/security.md]:
```suggestion
CircleCI Server customers can access the audit log feature from UI.
```

File: jekyll/_cci2/security.md [hunk 2]
```diff
@@ -59,9 +59,10 @@ A few different external services and technology integration points touch Circle
...
+※ Cloud customer needs to [contact CircleCI support](https://support.circleci.com/hc/en-us/requests/new) to request Audit log, and only an admin user of your organization is able to request this.
```
Comment (rosieyohannan) [inline jekyll/_cci2/security.md]:
```suggestion
Cloud customers can [contact CircleCI support](https://support.circleci.com/hc/en-us/requests/new) to request an Audit log. **Note:** only organization admin users can make an audit log request.
```

**Comment by rosieyohannan** [comment]
Thanks so much for this @ganezasan! Just a couple of editorial suggestions :-)

**Comment by ganezasan** [comment]
@rosieyohannan Thank you for your suggestions! I updated them 👍

File: jekyll/_cci2/security.md [hunk 3]
```diff
@@ -59,9 +59,10 @@ A few different external services and technology integration points touch Circle
 {% endraw %}
 
 ## Audit Logs
-The Audit Log feature is only available for CircleCI installed on your servers or private cloud. 
+CircleCI Server customers can access the audit log feature from UI.
```
Comment (rosieyohannan) [inline jekyll/_cci2/security.md]:
```suggestion
CircleCI Server customers can access the audit log feature from the UI.
```

Comment (rosieyohannan) [reply jekyll/_cci2/security.md]: Sorry I missed a word out the first time! Must have done that before I had any coffee :-)

Comment (ganezasan) [reply jekyll/_cci2/security.md]: NP, Thank you for your a lot of help 🎉

**Comment by ganezasan** [comment]
@rosieyohannan I updated the latest your suggestion! Please take a look!

**Pull request merged**